### DISCUSSION TITLE
How to save a model that was finetuned ?

### DISCUSSION THREAD

**Discussion by iR00i** [ThilinaRajapakse/simpletransformers#928] [Q&A] [answered]
hello i would like to save a model that i fine tuned in colab, i tried using this code, but it does not work
`model.save_model_args("bert.bin") `

**Comment by ThilinaRajapakse** [comment, marked as answer]
The model will be saved automatically depending on the `model_args` that you set. You can find the default values [here](https://simpletransformers.ai/docs/usage/#configuring-a-simple-transformers-model).

You can also find more information about the different ways to set up checkpoints/saving [here](https://simpletransformers.ai/docs/tips-and-tricks/#saving-checkpoints).

**Reply by iR00i** [reply]
ok thanks
